app/src/client/hooks/useColorMode.tsx

```diff
@@ -1,18 +1,64 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
```

Comment:
Do you still need change `useColorMode.tsx` and `DarkModeSwitcher.tsx` at all, as we agree the scope of change is mainly under packages/editor.